bindings/core/src/method_handler/wallet.rs

```diff
@@ -336,6 +336,24 @@ pub(crate) async fn call_wallet_method_internal(wallet: &Wallet, method: WalletM
                 .await?;
             Response::PreparedTransaction(data)
         }
+        WalletMethod::PrepareBeginStaking { params, options } => {
+            let data = wallet.prepare_begin_staking(params, options).await?;
+            Response::PreparedTransaction(data)
+        }
+        WalletMethod::PrepareExtendStaking {
+            account_id,
+            additional_epochs,
+        } => {
+            let data = wallet.prepare_extend_staking(account_id, additional_epochs).await?;
+            Response::PreparedTransaction(data)
+        }
+        WalletMethod::PrepareEndStaking { account_id } => {
+            let data = wallet.prepare_end_staking(account_id).await?;
```

Comment:
Note for myself, it's a bit weird that sometimes we use a variable and sometimes not. For such a small scope I wouldn't use one, like AnnounceCandidacy, but we can clean that up later.